components/com_joomgallery/views/upload/tmpl/default_ajax.php

```diff
@@ -7,259 +7,7 @@
-                  jQuery('#ajax_filecounter').val('');
-                }
-                jQuery('#ajax_imgtitle').removeClass('required');
-                jQuery('#ajax_imgtitle').removeAttr('aria-required').removeAttr('aria-invalid').removeAttr('required');
-                jQuery('#ajax_imgtitle-lbl').removeAttr('aria-invalid');
-                jQuery('#ajax_imgtitle').removeClass('invalid');
-                jQuery('#ajax_imgtitle-lbl').removeClass('invalid');
-                jQuery('#ajax_imgtitle').parent().parent().hide(750);
-                if(jQuery('#ajax_filecounter').length > 0 ) {
-                  jQuery('#ajax_filecounter').parent().parent().hide(750);
-                }
-                for(var i = 0; i < uploader._storedIds.length; i++) {
-                  var fileItemContainer = uploader.getItemByFileId(uploader._storedIds[i]);
-                  jQuery(fileItemContainer).find('.qq-upload-cancel').after('<input id="ajax_imgtitleid-' + uploader._storedIds[i] +'" class="qq-edit-imgtitle qq-editing" tabindex="0" type="text" value="" placeholder="' + '<?php echo JText::_('COM_JOOMGALLERY_COMMON_ENTER_IMAGE_TITLE', true); ?>' + '" required aria-required="true">');
-                  jQuery('#ajax_imgtitleid-' + uploader._storedIds[i]).change(function() {
-                    if(jQuery(this).val().trim() != '') {
-                      jQuery(this).removeClass('invalid').attr('aria-invalid', 'false');
-                    }
-                  });
-                }
-              }
-            });
-          }
-        });
-      </script>
-      <script type="text/template" id="qq-template">
-        <div class="qq-uploader-selector qq-uploader span12">
-          <div class="qq-upload-drop-area-selector qq-upload-drop-area span12" qq-hide-dropzone>
-            <span><?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_DRAGZONETEXT', true); ?></span>
-          </div>
-          <div class="qq-upload-button-selector qq-upload-button btn btn-large btn-success">
-            <div><i class="icon-plus icon-plus"></i> <?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_SELECT_IMAGES', true); ?></div>
-          </div>
-          <div class="small"><?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_DRAGNDROPHINT'); ?></div>
-          <span class="qq-drop-processing-selector qq-drop-processing">
-            <span><?php echo JText::_('COM_JOOMGALLERY_AJAXUPLOAD_DROPPROCESSINGTEXT', true); ?></span>
-            <span class="qq-drop-processing-spinner-selector qq-drop-processing-spinner"></span>
-          </span>
-          <ul class="qq-upload-list-selector qq-upload-list">
-            <li class="alert">
-              <div class="qq-progress-bar-container-selector">
-                <div class="qq-progress-bar-selector qq-progress-bar"></div>
-              </div>
-              <span class="qq-upload-spinner-selector qq-upload-spinner"></span>
-              <img class="qq-thumbnail-selector thumbnail" qq-max-size="50" qq-server-scale>
```

Comment:
It was set to false only in the MiniJoom, but the response parameter name for the frontend upload was wrong (see the change in the frontend controller). So, is the intention to set it to true only for the backend upload?